ewasm.md

```diff
@@ -276,7 +279,7 @@ From the executing account's storage, load the 32 bytes stored at the index spec
          <paramstack> INDEX : .ParamStack => .ParamStack </paramstack>
          <eeiK> . => EEI.getAccountStorage INDEX </eeiK>
 
-    rule <k> #waiting(eei.storageLoad) => #storeEeiResult(RESULTPTR, 32, VALUE) ... </k>
+    rule <k> #waiting(eei.storageLoad) => #storeEeiResult(RESULTPTR, 32, VALUE:Int) ... </k>
          <locals> ... 1 |-> <i32> RESULTPTR ... </locals>
          <eeiK> #result(VALUE) => . </eeiK>
```

Comment:
I think it makes more sense to assert sorts on the LHS of rules, so maybe put `VALUE:Int` in the `<eeiK>` cell, not the `<k>` cell. Your call.